digitaltwins/e2e/digitaltwin_telemetry.js

```diff
@@ -101,11 +101,11 @@ describe('Digital Twin Telemetry', function () {
       .then((client) => {
         debug('event hubs client: started');
         ehClient = client;
-        debug('registering digital twin client with test interfaceInstance');
+        debug('registering digital twin client with test component');
         return digitalTwinClient.register();
       }).then(function () {
         debug('digital twin client registered. sending telemetry: ' + testTelemetryBody);
-        return testInterfaceInstance.telemetry.send(testTelemetryBody);
+        return testComponent.telemetry.send(testTelemetryBody);
```

Comment:
>testComponent.telemetry.send [](start = 15, length = 28)

e2e still using the old component based functions rather than the digital twin client api.